DataModel/ontowikikul.ttl

```diff
@@ -14,6 +14,14 @@
 	rdfs:comment "Ontologia dla modelu danych wiki.kul.pl" ;
 	.
 
+ontowikikul:P3
+	a owl:AnnotationProperty ;
+	rdfs:label
+		"deanery (Latin Church)"@en ,
```

Comment:
Why _deanery_ is _owl:AnnotationProperty_??
See e.g. [ontohgis/administrative_type_150](https://onto.kul.pl/ontohgis/administrative_type_150.ttl):
```
ontohgis:administrative_type_150
        a                               owl:Class ;
        rdfs:label                      "decanatus"@la , "dekanat (Kościół katolicki ob. łacińskiego)"@pl , "deanery (Latin Church)"@en ;
```